frontend/institution/institutions.html

```diff
@@ -12,11 +12,19 @@ <h1>
                     </div>
                 </md-toolbar>
                 <md-card-content>
+                    <div style="margin-top: 5%;">
```

Comment:
This percentage in a vertical perspective will cause layout malfunction. Use a fixed length.